src/components/Nav/index.js

```diff
@@ -4,10 +4,14 @@ import PropTypes from 'prop-types'
 import LocalLink from '../LocalLink'
 
 import { logEvent } from '../../utils/ga'
+import { getFirstPage } from '../../utils/sidebar'
+
+const docsPage = getFirstPage()
```

Comment:
do we still need this? it should be always `dvc.org/doc`

Reply:
@pavelgrinchenko ? ^^

Reply:
I tried to remove this helper and rely on server.js redirects, but they work just for the initial load. For example, we load `/doc` page in the browser - in this case, server.js redirects us to the correct page. But if we are navigating from another page of dvc.org then Gatsby's router will intercept this request and show the 404 page, because in this case, it doesn't know about any redirects on the server side.
@iAdramelk told me that you are working on an index page for `/doc`. So we can wait for it and then remove this helper.
Another solution is to use something like [this plugin](https://www.gatsbyjs.org/packages/gatsby-plugin-client-side-redirect/), which will create all redirects on the client-side too, to have identical behavior on both sides: client and server.

Reply:
@pavelgrinchenko makes sense. it might be that we have a problem for other similar pages (we have 3 or 4 pages like this) if we have links to them in the docs?

> Another solution is to use something like this plugin, which will create all redirects on the client-side too, to have identical behavior on both sides: client and server.

I don't like the, but they might be good as a fallback for those sections we won't have index pages for some reason.